isodesign/process/calculation.py

```diff
@@ -194,30 +208,42 @@ def read_files(self, data):
             case ".mflux":
                 self._check_mflux(data, data_path.name)
 
-        self.data_dict.update({data_path.name : data})
+        self.data_dict.update({str(data_path): data})
+        
         # Add in dict_vmtf the files extensions without the dot as key and files names as value 
-        self.dict_vmtf.update({ext[1:]:data_path.stem})
-            
+        self.dict_vmtf.update({ext[1:]: data_path.stem})
+
+    def files_copy(self):
+        """
+        Method permit to copy the imported files in the linp folder. 
```

Comment:
Le début de cette phrase veut rien dire ^^
Je pense tu peux directement mettre "Copy the ..."